docs/Installation.rst

```diff
@@ -114,147 +116,271 @@ This is only required if the unit tests are to be build.
+
+#. Once `Homebrew` is installed, type the following command in your `Terminal` window. 
+   This will install the latest versions of all the packages required to correctly set 
+   up and run SCONE::
+
+     brew install gcc cmake python git openblas lapack libomp
+
+#. Close your `Terminal` window. Open a new `Finder` window and navigate to your `Home` directory 
+   (⌘ + ⇧ + h). Display hidden files (⌘ + ⇧ + .) and find the ``.zprofile`` file (this is the macOS 
+   equivalent of the ``.bashrc`` file on Linux distributions). Open it and insert **any of the 
+   following lines which are not already present** (note: this depends on whether you have a Mac 
+   running on an Intel CPU or an ARM - Apple Silicon - chip):
+   
+   * Intel::
+   
+          # Setting PATH for Python 3.13. The original version is saved in .zprofile.pysave.
+          PATH="/Library/Frameworks/Python.framework/Versions/3.13/bin:${PATH}"
+          export PATH
+
+          # Set shell environment for Homebrew.
+          eval "$(/usr/local/bin/brew shellenv)"
+
+          # Export pFUnit installation folder.
+          export PFUNIT_DIR=~/pFUnit/build/
+
+          # Export environmental variables required by pFUnit.
+          export F90=gfortran
+          export F90_VENDOR=GNU
+
+          # Export OpenMP root path and flags.
+          export OpenMP_ROOT=$(brew --prefix)/opt/libomp
+          export LDFLAGS="-L/usr/local/opt/libomp/lib"
+          export CPPFLAGS="-I/usr/local/opt/libomp/include"
+
+   * ARM::
+   
+          # Setting PATH for Python 3.13. The original version is saved in .zprofile.pysave.
+          PATH="/Library/Frameworks/Python.framework/Versions/3.13/bin:${PATH}"
+          export PATH
+
+          # Set shell environment for Homebrew.
+          eval "$(/opt/homebrew/bin/brew shellenv)"
+
+          # Export pFUnit installation folder.
+          export PFUNIT_DIR=~/pFUnit/build/
+
+          # Export environmental variables required by pFUnit.
+          export F90=gfortran
+          export F90_VENDOR=GNU
+
+          # Export OpenMP root path and flags.
+          export OpenMP_ROOT=$(brew --prefix)/opt/libomp
+          export LDFLAGS="-L/opt/homebrew/opt/libomp/lib"
+          export CPPFLAGS="-I/opt/homebrew/opt/libomp/include"
+
+#. Save the changes you made in your ``.zprofile`` file and close it. 
+   You may now hide hidden files (⌘ + ⇧ + .).
+
+#. Open a new `Terminal` window. By default, it should open in your `Home` directory, 
+   but if not navigate to it by entering::
+
+     cd
+
+#. Download the pFUnit repository from Git, enter the source code repository and 
+   create a build directory (e.g. build) by typing the following commands::
+
+	git clone https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
+	cd pFUnit
+	mkdir build
+	cd build
+
+#. Before proceeding, **make sure that the default C compiler is Apple Clang by entering 
+   the following command**::
+
+     gcc --version
+   
```

Comment:
I don't follow this part. If we verify `clang` why do we want to invoke `gcc`? Is this a typo?